### PR TITLE
feat(i18n): fix typos and improve translations in Spanish

### LINKED ISSUE #384: [FEATURE] i18n: Improve Spanish Translations
# 🚀 Feature request

First of all I want to thank you for this incredible library! I've been using it in the last project I'm working on and it's perfect!

### Is your feature request related to a problem?

 I've discovered typos in the Spanish translations strings and some of them can be improved.

### Describe the solution you'd like

 

_in `taiga-ui/projects/i18n/languages/spanish/addon-editor.ts`_

- `italic`: Can be improved to '**Cursiva**'.

_In `taiga-ui/projects/i18n/languages/spanish/core.ts`_

- `defaultErrorMessage`: The translation should be 'El valor es **inválido**'.
- `shortWeekDays`, `months`: In Spanish the weekdays and months aren't capitalized since they aren't proper nouns. This can be improved in order to adhere to Angular's default translation strings and most of the 3rd party libraries.

_In `taiga-ui/projects/i18n/languages/spanish/kit.ts`_
- `passwordTexts`: There is a typo in '**Muetra**', it should be '**Muestra**', also '**clave**' can be changed for '**contraseña**' since it is more often used in web forms. Both can be improved to ['**Mostrar contraseña**', '**Ocultar contraseña**'] to make the verbs in the infinitive form.
- `shortCalendarMonths`: Same as in `core.ts`.
- `fileTexts.preview`: It should be '**Previsualizar**'.
- `fileTexts.loadingError`: Can be improved to 'Error de **carga**'.
- `inputFileTexts.formatRejectionReason`: It should be 'Formato de **archivo** erróneo | inválido'

### Describe alternatives you've considered

I can submit a PR with the changes if that's okay for you! 😄🧑🏼‍💻

### PR DESCRIPTION
**Pull request by samosunaz**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [x] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Some translation strings in Spanish have typos and others have been translated wrong.

Closes #384 

## What is the new behavior?

Typos have been fixed and improved some translations.

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/i18n/languages/spanish/core.ts [hunk 1]
```diff
@@ -2,22 +2,22 @@ import {LanguageCore} from '@taiga-ui/i18n/interfaces';
...
+        'noviembre',
+        'diciembre',
     ],
     close: 'Cerrar',
     nothingFoundMessage: 'Nada encontrado',
-    defaultErrorMessage: 'El valor es válido',
+    defaultErrorMessage: 'El valor es inválido',
     spinTexts: ['Previo', 'Siguiente'],
-    shortWeekDays: ['Lun', 'Mar', 'Mie', 'Jue', 'Vie', 'Sáb', 'Dom'],
+    shortWeekDays: ['lun', 'mar', 'mié', 'jue', 'vie', 'sáb', 'dom'],
```
Comment (waterplea) [inline projects/i18n/languages/spanish/core.ts:22]: Does it make more sense to use week days and months with small letter? They appear in calendars and look good with capital letter in Russian and English.

**Review by waterplea — APPROVED** [review]

**Review by MarsiBarsi — APPROVED** [review]
Hey! Thank you very much for your PR :) 

If you don't mind, please, add yourself in .github/CODEOWNERS as an expert in Spanish (replace `# TODO: find an expert in Spanish` with `projects/i18n/languages/turkish/* @samosunaz`)

File: projects/i18n/languages/spanish/core.ts (hunk 1 above)
Comment (samosunaz) [reply projects/i18n/languages/spanish/core.ts:22]: To be honest, it is also looks better for me, if that's for the calendar component I don't see any problem using capitalization, but for formatting (for ex. '**next Monday**' = '**siguiente lunes**') the guideline is to use lowercase. 
I'm going to revert that change

**Comment by samosunaz** [comment]
> Hey! Thank you very much for your PR :)
> 
> If you don't mind, please, add yourself in .github/CODEOWNERS as an expert in Spanish (replace `# TODO: find an expert in Spanish` with `projects/i18n/languages/spanish/* @samosunaz`)

Of course! I don't mind doing that 😄

**Review by MarsiBarsi — APPROVED** [review]
Thank you again!

**Pull request merged**